### PR TITLE
chore: use plugin rather than plug-in

### PR DESCRIPTION
**Pull request by themr0c**
## What does this pull request change?

chore: use plugin rather than plug-in due to style-guide change

## What issues does this pull request fix or reference?

fixes https://issues.redhat.com/browse/RHDEVDOCS-4763

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/creating-a-telemetry-plugin.adoc [hunk 1]
```diff
@@ -377,15 +377,15 @@ components:
 
 
 .Verification steps
-. Verify that the `telemetry-plug-in` container is running in the {devworkspace} pod. Here, this is verified by checking the Workspace view within the editor.
+. Verify that the `telemetry-plugin` container is running in the {devworkspace} pod. Here, this is verified by checking the Workspace view within the editor.
```
Comment (max-cx) [inline modules/administration-guide/pages/creating-a-telemetry-plugin.adoc]: Is the container name an arbitrary name or as specified by the devs?

File: modules/administration-guide/pages/the-woopra-telemetry-plugin.adoc [hunk 2]
```diff
@@ -1,24 +1,24 @@
 :_content-type: PROCEDURE
-:description: The Woopra telemetry plug-in
+:description: The Woopra telemetry plugin
 :keywords: extensions, telemetry
-:navtitle: The Woopra telemetry plug-in
-:page-aliases: extensions:the-woopra-telemetry-plug-in.adoc
+:navtitle: The Woopra telemetry plugin
+:page-aliases: extensions:the-woopra-telemetry-plugin.adoc
```
Comment (max-cx) [inline modules/administration-guide/pages/the-woopra-telemetry-plugin.adoc]: Are you sure you want to change this line?

File: modules/end-user-guide/pages/troubleshooting-slow-workspaces.adoc [hunk 3]
```diff
@@ -75,20 +75,20 @@ apiVersion: 1.0.0
 components:
   -
     type: chePlugin
-    id: id/of/plug-in
+    id: id/of/plugin
```
Comment (max-cx) [inline modules/end-user-guide/pages/troubleshooting-slow-workspaces.adoc]:
```suggestion
    id: __<plugin_id>__
```
Prob also need to update the attributes for this block?

File: modules/end-user-guide/pages/troubleshooting-slow-workspaces.adoc [hunk 4]
```diff
@@ -114,8 +114,8 @@ apiVersion: 1.0.0
...
```
Comment (max-cx) [inline modules/end-user-guide/pages/troubleshooting-slow-workspaces.adoc]:
```suggestion
    id: __<plugin_id>__
```
Prob also need to update the attributes for this block?

**Review by azatsarynnyy — APPROVED** [review]

File: modules/administration-guide/pages/configuring-observability.adoc [hunk 5]
```diff
@@ -2,7 +2,7 @@
 :description: Configuring observability
 :keywords: administration-guide, configuring, observability, logs, monitoring
 :navtitle: Configuring observability
-:page-aliases: .:retrieving-che-logs.adoc, .:viewing-kubernetes-events.adoc, viewing-kubernetes-events.adoc, .:viewing-plug-in-broker-logs.adoc, viewing-plug-in-broker-logs.adoc, retrieving-che-logs.adoc, viewing-che-server-logs.adoc, viewing-external-service-logs.adoc
+:page-aliases: .:retrieving-che-logs.adoc, .:viewing-kubernetes-events.adoc, viewing-kubernetes-events.adoc, .:viewing-plugin-broker-logs.adoc, viewing-plugin-broker-logs.adoc, retrieving-che-logs.adoc, viewing-che-server-logs.adoc, viewing-external-service-logs.adoc
```
Comment (themr0c) [inline modules/administration-guide/pages/configuring-observability.adoc]:
```suggestion
:page-aliases: .:retrieving-che-logs.adoc, .:viewing-kubernetes-events.adoc, viewing-kubernetes-events.adoc, .:viewing-plug-in-broker-logs.adoc, viewing-plug-in-broker-logs.adoc, retrieving-che-logs.adoc, viewing-che-server-logs.adoc, viewing-external-service-logs.adoc
```

File: modules/administration-guide/pages/creating-a-telemetry-plugin.adoc [hunk 6]
```diff
@@ -1,11 +1,11 @@
 :_content-type: REFERENCE
-:description: Creating a telemetry plug-in
+:description: Creating a telemetry plugin
 :keywords: extensions, telemetry
-:navtitle: Creating a telemetry plug-in
-:page-aliases: extensions:creating-a-telemetry-plug-in.adoc
+:navtitle: Creating a telemetry plugin
+:page-aliases: extensions:creating-a-telemetry-plugin.adoc
```
Comment (themr0c) [inline modules/administration-guide/pages/creating-a-telemetry-plugin.adoc]:
```suggestion
:page-aliases: extensions:creating-a-telemetry-plug-in.adoc
```

File: modules/administration-guide/pages/the-woopra-telemetry-plugin.adoc (hunk 2 above)
Comment (themr0c) [inline modules/administration-guide/pages/the-woopra-telemetry-plugin.adoc]:
```suggestion
:page-aliases: extensions:the-woopra-telemetry-plug-in.adoc
```

Comment (themr0c) [reply modules/administration-guide/pages/the-woopra-telemetry-plugin.adoc]: Mistake, indeed

File: modules/administration-guide/pages/creating-a-telemetry-plugin.adoc (hunk 1 above)
Comment (themr0c) [inline modules/administration-guide/pages/creating-a-telemetry-plugin.adoc]:
```suggestion
. Verify that the telemetry plugin container is running in the {devworkspace} pod. Here, this is verified by checking the Workspace view within the editor.
```

Comment (themr0c) [reply modules/administration-guide/pages/creating-a-telemetry-plugin.adoc]: yes

File: modules/end-user-guide/pages/customizing-workspace-components.adoc [hunk 7]
```diff
@@ -2,7 +2,7 @@
...
+:page-aliases: configuring-a-workspace-using-a-devfile.adoc, making-a-workspace-portable-using-a-devfile.adoc, authoring-devfiles-version-1.adoc, authoring-devfiles-version-2.adoc, authoring-devfiles.adoc,  defining-custom-commands-for-che-theia.adoc, adding-a-vs-code-extension-to-a-workspace.adoc, adding-a-vs-code-extension-to-the-che-plugin-registry.adoc, adding-tools-to-che-after-creating-a-workspace.adoc, using-private-container-registries.adoc, customizing-developer-environments.adoc, what-is-a-che-theia-plugin.adoc, testing-a-visual-studio-code-extension-in-che.adoc, publishing-metadata-for-a-vs-code-extension.adoc, contributor-guide:developing-che-theia-plugins.adoc, contributor-guide:testing-che-theia-plugins.adoc, contributor-guide:publishing-che-theia-plugins.adoc, contributor-guide:adding-support-for-a-new-language.adoc, contributor-guide:adding-support-for-a-new-debugger.adoc, contributor-guide:che-extensibility-reference.adoc, contributor-guide:che-extension-points.adoc, contributor-guide:che-theia-plugin-api.adoc, contributor-guide:debug-adapter-protocol.adoc, contributor-guide:language-server-protocol.adoc, customizing-workspaces-components.adoc
```
Comment (themr0c) [inline modules/end-user-guide/pages/customizing-workspace-components.adoc]:
```suggestion
:page-aliases: configuring-a-workspace-using-a-devfile.adoc, making-a-workspace-portable-using-a-devfile.adoc, authoring-devfiles-version-1.adoc, authoring-devfiles-version-2.adoc, authoring-devfiles.adoc,  defining-custom-commands-for-che-theia.adoc, adding-a-vs-code-extension-to-a-workspace.adoc, adding-a-vs-code-extension-to-the-che-plugin-registry.adoc, adding-tools-to-che-after-creating-a-workspace.adoc, using-private-container-registries.adoc, customizing-developer-environments.adoc, what-is-a-che-theia-plug-in.adoc, testing-a-visual-studio-code-extension-in-che.adoc, publishing-metadata-for-a-vs-code-extension.adoc, contributor-guide:developing-che-theia-plug-ins.adoc, contributor-guide:testing-che-theia-plug-ins.adoc, contributor-guide:publishing-che-theia-plug-ins.adoc, contributor-guide:adding-support-for-a-new-language.adoc, contributor-guide:adding-support-for-a-new-debugger.adoc, contributor-guide:che-extensibility-reference.adoc, contributor-guide:che-extension-points.adoc, contributor-guide:che-theia-plug-in-api.adoc, contributor-guide:debug-adapter-protocol.adoc, contributor-guide:language-server-protocol.adoc, customizing-workspaces-components.adoc
```

File: modules/overview/pages/introduction-to-eclipse-che.adoc [hunk 8]
```diff
@@ -120,18 +120,18 @@ For situations where the default IDE does not cover the use cases of the users o
 
 {prod} is extensible in different ways:
 
-* *Plug-ins* to add capabilities to the IDE. Che-Theia plug-ins rely on APIs compatible with Visual Studio Code. Plug-ins are isolated and provide their own dependencies packaged in containers.
+* *Plug-ins* to add capabilities to the IDE. Che-Theia plugins rely on APIs compatible with Visual Studio Code. Plug-ins are isolated and provide their own dependencies packaged in containers.
```
Comment (max-cx) [inline modules/overview/pages/introduction-to-eclipse-che.adoc:123]: Remove this line?

Comment (themr0c) [reply modules/overview/pages/introduction-to-eclipse-che.adoc:123]: That's the task @deerskindoll is currently working on, I will stay on scope.

Comment (max-cx) [reply modules/overview/pages/introduction-to-eclipse-che.adoc:123]: Ah, apologies then.

**Review by max-cx — APPROVED** [review]

**Pull request merged**